src/main/java/nextstep/subway/utils/ValidationUtils.java

```diff
@@ -0,0 +1,13 @@
+package nextstep.subway.utils;
+
+public class ValidationUtils {
+
+    public static boolean isEmpty(String str) {
+        return isNull(str) || str.length() == 0;
```

Comment:
```suggestion
        return Objects.isNull(str) || str.isEmpty();
```
이런 식으로 표현할 수 있습니다 :)